docs/configure-custom-values-envoy.md

```diff
@@ -161,6 +134,21 @@ envoy:
     allowPrivilegeEscalation: false
 ```
 
+### Image configurations (Default value is recommended)
+
+If you want to change the image repository and version, you can use `envoy.image.repository`. Please specify the container repository information that you pull the Scalar Envoy container image.
+
+```yaml
+envoy:
+  image:
+    repository: <Container image of Scalar Envoy>
+```
+
+If you use AWS/Azure Marketplace, please refer to the following documents for more details.
+
+* [How to install Scalar products through AWS Marketplace](https://github.com/scalar-labs/scalar-kubernetes/blob/master/docs/AwsMarketplaceGuide.md)
+* [How to install Scalar products through Azure Marketplace](https://github.com/scalar-labs/scalar-kubernetes/blob/master/docs/AzureMarketplaceGuide.md)
+
```

Comment:
I moved this section from `Recommended in the production environment` to `Default value is recommended`.
We provide the Scalar Envoy container image on the public container registry. So, basically, users don't need to change this configuration.

Also, I removed the `tag` configuration. This is because we recommend using the default version that is set on the Helm Chart side.